terraform/dev/main.tf

```diff
@@ -17,29 +17,36 @@ terraform {
 }
 
 module "gke" {
-  source = "git::git@github.com:cloudkite-io/terraform-modules.git//modules/gke?ref=v0.0.1"
+  source = "git::git@github.com:cloudkite-io/terraform-modules.git//modules/gcp/gke?ref=v0.0.1"
   environment = var.environment
   gcp = var.gcp
   gke = var.gke
   gke_nodepools = var.gke_nodepools
   gke_pods_secondary_range_name = module.vpc.gke_subnetwork_secondary_range_name_services
-  gke_service_account = module.gke_service_account.email
   gke_services_secondary_range_name = module.vpc.gke_subnetwork_secondary_range_name_pods
   network = module.vpc.network
   subnetwork = module.vpc.gke_subnetwork
+  enable_network_policy = true
```

Comment:
Our modules should set sane defaults. If this is always going to be true, then just remove the option and make it part of the module's defaults.

Reply:
I have removed the velero chart and defaulted the enable_network_policy to true in the GKE module.